cypress/support/e2e.ts

```diff
@@ -87,6 +87,14 @@ beforeEach(() => {
     { fixture: 'toggle-enable-redirect-1963.json' }
   ).as('getFeatureToggleRedirect1963')
 
+  cy.intercept(
```

Comment:
Jeg kommer på at vi ikke har opprettet noe Jira sak for å utvide Testcase for offentlig tjenestepensjon. Har du mulighet til å gjøre det og involvere Øyvind i det?